blueprints/app/files/package.json

```diff
@@ -12,17 +12,17 @@
   },
   "scripts": {
     "build": "ember build --environment=production",
-    "lint": "concurrently \"npm:lint:*(!fix)\" --names \"lint:\"",
+    "lint": "concurrently --group --raw \"npm:lint:*(!fix)\"",
     "lint:css": "stylelint \"**/*.css\"",
-    "lint:css:fix": "concurrently \"npm:lint:css -- --fix\"",
-    "lint:fix": "concurrently \"npm:lint:*:fix\" --names \"fix:\"",
+    "lint:css:fix": "stylelint \"**/*.css\" --fix",
```

Comment:
Avoid nested `concurrently` invocations. This makes the output more readable.
Repeating the glob seems like an acceptable trade off, as it's not something that will change often. Reading / looking at the output will happen far more often.